index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+	<head>
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<title>Project1</title>
+		<script src="https://kit.fontawesome.com/62fc5eef3c.js" crossorigin="anonymous"></script>
+	</head>
+	<body>
+		<!--Navbar -->
+		<header>
+			<nav>
+				<div class="nav-btns-left">
+					<p><a class="link" href="#about">About</a></p>
+					<p><a class="link" href="#portfolio">Portfolio</a></p>
+				</div>
+				<div class="nav-btn-right">
+						<p><a class="link" href="#contact">Contact</a></p>
+				</div>
+			</nav>
+		</header>
+	<!-- Main -->
+	 <main>
+		<div class="imgs-container">
+			<img src="kid.jpg" alt="kid statue">
+			<img src="london.jpg" alt="london eye image">
+			<img src="girl.jpg" alt="girl relaxing">
+			<img src="statue.jpg" alt="image of a statue">
+		</div>
+		<section class="hello-section">
+			<h1>Hello there!</h1>
+			<h2>Welcome to Cosmin's portfolio</h2>
+		</section>
+	</main>
+	<!--About section -->
+	<section class="about-section">
+		<h2 class="h3-about" id="about">About</h3>
+		<h3>Hi, I’m Cosmin , originally from Romania..I chose to come to the UK roughly 4 years ago to start a new chapter. Having just finished my master's degree in Geography in Romania, the oportunities to start something there after I finished my college, were quite limited. My life in the UK was like a rollercoaster in all this years, with a pandemy just gone..and kept switching between jobs ..I found myself now to be a student at iO academy, and I must admit I absolutely love it! I just hope that I will be able soon to fullfill my dream and become a programmer.
+		<!--Portfolio section -->
+	</section>
+	<section class="portfolio-section">
+		<div class="portfolio-description">
+			<h2 class="h2-portfolio" id="portfolio">Portfolio</h2>
+			<h3 class>Here are some of my photographs</h3>
+		</div>
+		<div class="card">
+			<h3>Day dreaming</h3>
+			<div class="card-img-text">
+				<img src="girl.jpg" alt="girl relaxing">
+				<p>Sometimes..I just like to grab my camera and go somewhere, and try to capture random stuff but also something which will have a sense of uniqueness in it, and, street photography it's one of the best things to do such thing. In one of the days, I just woke up and I said will go around Bristol Harbour and try to capture something interesting. I took quite few shots, of people, seaguls, boats everything...but the image which fascinated me the most was this capture of this girl..who..was just relaxing and day dreaming, the atmosphere around the city was very relaxed in that day, weather was nice..and this image just transmitted me the vibe of that day</p>
+			</div>
+		</div>
```

Comment:
since this should be coding projects in the future, are you sure you don't want to put links? e.g. a link to github?